src/flaskr/home.py

```diff
@@ -17,6 +22,10 @@ def home():
         field_id = request.form.get('field_id')
         start_date = request.form.get('start_date')
         time_increment = request.form.get('time_increment')
+        is_cleanse = request.form.get('cleanse')
+        is_predict = request.form.get('predict')
+        prediction_date = request.form.get('prediction_date')
+        chart_type = request.form.get('chartType')  # Retrieve the selected chart type
```

Comment:
maybe make it more robust by using like a dropdown menu if possible becuase if they misspell it itll cause an error